### DISCUSSION TITLE
What's the difference between ActorRef.cast and ActorRef.send_message?

### DISCUSSION THREAD

**Discussion by phil-skillwon** [slawlor/ractor#157] [Q&A] [answered]
like this:
actor.cast(MyMsg { inner: "123456".to_owned() })?;
actor.send_message(MyMsg { inner: "7654321...".to_owned() })?;

**Comment by slawlor** [comment, marked as answer]
There's no difference really. Cast is an alias of send_message.

The reason it exists as cast is to match Erlang OTP syntax.
